### DISCUSSION TITLE
Please allow adding any issue to a project via the GraphQL API

### DISCUSSION THREAD

**Discussion by hexagonrecursion** [community/community#9764] [Projects and Issues]
I can manually add issues from any repository to my projects by pasting the issue URL into the project. This is useful for managing my contributions to repositories I do not own. Unfortunately when I try doing the same via the GraphQL API explorer I get:

```json
{
  "data": {
    "addProjectNextItem": null
  },
  "errors": [
    {
      "type": "VALIDATION",
      "path": [
        "addProjectNextItem"
      ],
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "message": "This project item doesn't belong to correct owner"
    }
  ]
}
```

```graphql
mutation {
  addProjectNextItem(input: {projectId: "PN_kwHOAyLyIs4AASiD", contentId: "MDU6SXNzdWU4MTIxMDkxMg=="}) {
    clientMutationId
  }
}
```

**Comment by aeneasr** [comment]
I was hit by this as well as I was trying this out. It would really be of great help for us to be able to add any issue to a project board. We need this as we are handling several github organizations  at [Ory](https://github.com/ory)

**Comment by michael-hawker** [comment]
Yeah, it'd be nice even to just have the ability to add a repository and some filter conditions (like a search query and/or label) and be able to have new issues that match that be added to the project table/board view automatically.
